### PR TITLE
Blog updates

### PR DESCRIPTION
**Pull request by balapriyac**

### CODE REVIEW

File: blog/_posts/2023-02-16-psycopg2-postgres-python.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ In practice, you should store the details of the database and the credentials re
 
 **But why is this helpful?** When working on a project, you may need to connect to multiple databases or connect to a database from more than one module. In such cases, if the database credentials change, you can only modify the contents of the config file *without* introducing breaking changes elsewhere.
 
-Also, exposing sensitive info like passwords in source code is a security risk. So you should always store the credentials in a config file or set them as [environment variables](/blog/bash-variables) whose values you can fetch as needed.
```
Comment (adamgordonbell) [inline blog/_posts/2023-02-16-psycopg2-postgres-python.md:89]: Part of the reason for these links is for SEO. We shouldn't use them if they don't make sense. And many of the ones you've found done.  
But I think this one does if we make it more specific:
https://earthly.dev/blog/bash-variables/#environmental-variables

**Review by adamgordonbell — APPROVED** [review]

File: blog/_posts/2023-02-16-psycopg2-postgres-python.md (hunk 1 above)
Comment (balapriyac) [reply blog/_posts/2023-02-16-psycopg2-postgres-python.md:89]: > Part of the reason for these links is for SEO. We shouldn't use them if they don't make sense. And many of the ones you've found don't seem to make sense. But I think this one does if we make it more specific: https://earthly.dev/blog/bash-variables/#environmental-variables

Thank you. Yes, that makes sense. I'll add that env. vars link to the subprocess article, too. Last week, we updated the editing checklist to ask editors to add relevant internal links going forward. That way, Sameer won't have to run `link-opp` to get internal link recommendations.

**Comment by jalletto** [comment]
@balapriyac just want to confirm this is ready to merge.

**Comment by balapriyac** [comment]
> @balapriyac just want to confirm this is ready to merge.

@jalletto yes, this is ready to merge.

**Comment by jalletto** [comment]
@balapriyac There are some conflicts that need to be resolved before this can be merged.

**Pull request closed without merging**